api/views/teledeclaration.py

```diff
@@ -56,20 +64,23 @@ def validateDiagnostic(diagnostic):
             raise ValidationError("Données d'approvisionnement manquantes")
 
 
+@extend_schema_view(
+    post=extend_schema(
+        summary="Annuler une télédéclaration existante.",
+        description="En annulant une télédéclaration existante l'utilisateur devra en créer une nouvelle une fois que le diagnostic a été corrigé.",
```

Comment:
```suggestion
        description="En annulant une télédéclaration existante, on regagne l'option de modifier le diagnostic. L'utilisateur devra créer une nouvelle télédéclaration a part pour télédéclarer le diagnostic à nouveau.",
```
Pas exactement mais je veux souligner la raison pour laquelle qqn voudrait annuler une télédéclaration, et que c'est pas obligatoire de le re-télédéclarer, mais nécessaire pour revenir au état complèt

Reply:
Je reformulerai dans un commit :+1:

Reply:
https://github.com/betagouv/ma-cantine/pull/1926/commits/2a73e47f4555205e17ac4918d224f1831b214a68